tests/integ/test_df_to_snowpark_pandas.py

```diff
@@ -47,9 +49,14 @@ def test_to_snowpark_pandas_no_modin(session, tmp_table_basic):
         # TODO: SNOW-1552497: after upgrading to modin 0.30.1, Snowpark pandas will support
         # all pandas 2.2.x, and this function call will raise a ModuleNotFoundError since
         # modin is not installed.
+        match = (
+            "Snowpark pandas does not support Python 3.8. Please update to Python 3.9 or later"
+            if sys.version_info.major == 3 and sys.version_info.minor == 8
```

Comment:
Do we care about message for earlier versions of python, ie. < 3.8 ?

Reply:
Within Snowpark pandas, we currently check if the version ==3.8. Older Python versions have been EOL'd anyway, and even if a user somehow manages to set up a Snowpark (pandas or non-pandas) environment, they would encounter an error when trying to import pandas anyway since pandas 2.1+ only works with Python 3.9 and later.